### ISSUE TITLE
yify changed to yts.ag

### ISSUE THREAD

**Issue by glenb77** [jmarth/plugin.video.kmediatorrent#142]
someone else has made a clone site of former yify. and releasing current movies. site looks exactly the same at yts.ag, but when i enter into settings under movies, no titles show up. i amon 2.37. can this plugin be updated to use the new site ?

**Comment by theboy181** [comment]
What's the site adress?

**Comment by mancuniancol** [comment]
https://yts.ag/

**Comment by C-Fu** [comment]
Yts.ag is not from the same guys. So can't expect the site to work the
same, even if it may look the same. Best to be careful though.

**Comment by mancuniancol** [comment]
You are right, it is not the same guys.  Actually, it is the same group that hijack eztv.  I've done a viewer that allow to go to the site and use kmediatorrent meanwhile the site is implemented in kmediatorrent.

It is called  YTS VIEWER.  You can install from this repo:  https://github.com/mancuniancol/repository.providerspulsarunofficial/archive/1.0.6.zip

**Comment by sanhajisoufiane** [comment]
thanks man I've been looking for a work around for quite some time .. now I can enjoy my 3d movies again👌👍✌

**Comment by sanhajisoufiane** [comment]
Can you plz make one for cpasbien for french people it would be really awesome thk any way for the other viewers i know its a lot of work .

**Comment by mancuniancol** [comment]
Bien sûr ;)

**Comment by sanhajisoufiane** [comment]
Thkx a lot man....merci 😊

**Comment by mancuniancol** [comment]
done!

**Comment by sanhajisoufiane** [comment]
Flash!!!your great man thanks again

**Comment by theboy181** [comment]
This repo no longer has the viewer.. any ideas? the 1.2 version is says dependency not met when trying to install from the installer from the 17th..

**Comment by mancuniancol** [comment]
Try this one 
https://github.com/mancuniancol/repository.providerspulsarunofficial/archive/1.0.6.zip

**Comment by theboy181** [comment]
I tried downloading the zip files from git for the viewers that were updated Yesterday. None of them are working. They all just curl then time out. Will not show any of the links. 

Yts 1.3 did work on my laptop but I'm not sure how. L337 wouldn't neither did torrentz. 

My htc will not work with a fresh install of the plugins.

**Comment by mancuniancol** [comment]
It is better from the repo. But. I don't want to hijack this thread. We can continue the conversation here:
http://forums.tvaddons.ag/addon-releases/37973-torrent-rss-kickass-yts-torrentz-viewers.html
